From e57d5ba73cc3f420669b49bc047bb51a6f70680d Mon Sep 17 00:00:00 2001
From: Mark Enriquez <enriquezmark36@gmail.com>
Date: Sat, 4 Aug 2018 16:18:33 +0800
Subject: hwcomposer: kanas: define HWC_SUPPORT

This is missing, causing the query() to always have a value of 0.

Change-Id: Iac7889fe8166c9d7018ee943c3d9b384a334b36b
Signed-off-by: Mark Enriquez <enriquezmark36@gmail.com>
---
 hwcomposer/scx15/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwcomposer/scx15/Android.mk b/hwcomposer/scx15/Android.mk
index d909f14..c48cb85 100644
--- a/hwcomposer/scx15/Android.mk
+++ b/hwcomposer/scx15/Android.mk
@@ -137,6 +137,7 @@ endif
 
 ifeq ($(strip $(SOC_SCX35)),true)
 LOCAL_CFLAGS += -DGSP_ADDR_TYPE_PHY
+LOCAL_CFLAGS += -DHWC_SUPPORT
 endif
 
 endif # DEVICE_WITH_GSP
-- 
2.19.1

